scheduler/src/cook/mesos/mesos_compute_cluster.clj

```diff
@@ -292,7 +292,7 @@
     container-defaults))
 
 ; Internal method
```

Comment:
This comment appears to be outdated now?

Reply:
It is still an internal method. Only had to do this for testing purposes.